user-info-api/src/main/java/de/intension/api/UserInfoAttribute.java

```diff
@@ -0,0 +1,83 @@
+package de.intension.api;
+
+import de.intension.api.enumerations.OrganisationsTyp;
+import de.intension.api.enumerations.PersonenStatus;
+import de.intension.api.enumerations.Vertrauensstufe;
+
+public enum UserInfoAttribute
+{
+
+    HEIMATORGANISATION_ID(UserInfoAttributeNames.HEIMATORGANISATION_ID, UserInfoAttributeNames.HEIMATORGANISATION_ID, Boolean.TRUE, null),
+    HEIMATORGANISATION_NAME(UserInfoAttributeNames.HEIMATORGANISATION_NAME, UserInfoAttributeNames.HEIMATORGANISATION_NAME, Boolean.TRUE, null),
+    HEIMATORGANISATION_BUNDESLAND(UserInfoAttributeNames.HEIMATORGANISATION_BUNDESLAND, UserInfoAttributeNames.HEIMATORGANISATION_BUNDESLAND, Boolean.TRUE,
+            null),
+    PERSON_FAMILIENNAME(UserInfoAttributeNames.PERSON_FAMILIENNAME, UserInfoAttributeNames.PERSON_FAMILIENNAME, Boolean.FALSE, null),
+    PERSON_FAMILIENNAME_INITIALEN(UserInfoAttributeNames.PERSON_FAMILIENNAME_INITIALEN, UserInfoAttributeNames.PERSON_FAMILIENNAME_INITIALEN, Boolean.FALSE,
+            null),
+    PERSON_VORNAME(UserInfoAttributeNames.PERSON_VORNAME, UserInfoAttributeNames.PERSON_VORNAME, Boolean.FALSE, null),
+    PERSON_VORNAME_INITIALEN(UserInfoAttributeNames.PERSON_VORNAME_INITIALEN, UserInfoAttributeNames.PERSON_VORNAME_INITIALEN, Boolean.FALSE, null),
+    PERSON_AKRONYM(UserInfoAttributeNames.PERSON_AKRONYM, UserInfoAttributeNames.PERSON_AKRONYM, Boolean.FALSE, null),
+    PERSON_GEBURTSDATUM(UserInfoAttributeNames.PERSON_GEBURTSDATUM, UserInfoAttributeNames.PERSON_GEBURTSDATUM, Boolean.FALSE, null),
+    PERSON_ALTER(UserInfoAttributeNames.PERSON_ALTER, UserInfoAttributeNames.PERSON_ALTER, Boolean.FALSE, null),
+    PERSON_GESCHLECHT(UserInfoAttributeNames.PERSON_GESCHLECHT, UserInfoAttributeNames.PERSON_GESCHLECHT, Boolean.FALSE, null),
+    PERSON_LOKALISIERUNG(UserInfoAttributeNames.PERSON_LOKALISIERUNG, UserInfoAttributeNames.PERSON_LOKALISIERUNG, Boolean.TRUE, "de-DE"),
+    PERSON_VERTRAUENSSTUFE(UserInfoAttributeNames.PERSON_VERTRAUENSSTUFE, UserInfoAttributeNames.PERSON_VERTRAUENSSTUFE, Boolean.TRUE, Vertrauensstufe.VOLL),
+
+    PERSON_KONTEXT_ID(UserInfoAttributeNames.PERSON_KONTEXT_ID, UserInfoAttributeNames.PERSON_KONTEXT_ID, Boolean.TRUE, Vertrauensstufe.VOLL),
+    PERSON_KONTEXT_ORG_VIDIS_ID(UserInfoAttributeNames.PERSON_KONTEXT_ORG_VIDIS_ID, UserInfoAttributeNames.PERSON_KONTEXT_ORG_VIDIS_ID, Boolean.TRUE, null),
```

Comment:
A tad long maybe :shrug: 
![image](https://user-images.githubusercontent.com/42644709/207824507-1d820882-da63-45db-b875-eb6656ca46b9.png)

Reply:
this is an issue with the Keycloak 20 GUI. In 16.1.1 it was fine.
We can life with it...

Reply:
True - noone likes the keycloak.v2 admin UI :grin: